stable/stackdriver-exporter/values.yaml

```diff
@@ -83,3 +83,12 @@ serviceAccount:
   # If not set and create is false, 'default' is used
   # If not set and create is true, a name is generated using the fullname template
   name:
+
+# Enable this if you're using https://github.com/coreos/prometheus-operator
+serviceMonitor:
+  enabled: false
+  namespace: monitoring
+  # fallback to the prometheus default unless specified
+  # interval: 10s
+  ## Defaults to what's used if you follow CoreOS [Prometheus Install Instructions](https://github.com/helm/charts/tree/master/stable/prometheus-operator#tldr)
+  honorLabels: true
```

Comment:
Same as above.

Reply:
The only other comment i could find was about `{{- end }}`, am I missing something?

Reply:
I mean there may need a new line.

Reply:
@Aisuko sorry think I am misreading something, there is already a newline after this line